test/integration/good/compiler-optimizations/cppO1.expected

```diff
@@ -6351,7 +6355,7 @@ class expr_prop_fail2_model final : public model_base_crtp<expr_prop_fail2_model
               emit_transformed_parameters = true, const bool
               emit_generated_quantities = true, std::ostream*
               pstream = nullptr) const {
-    const size_t num_params__ = ((1 + J) + 1);
+    const size_t num_params__ = ((1 + (J * 1)) + 1);
```

Comment:
Where is the `* 1` coming from?

Reply:
Oh from tuples size. Can we be clever and not do * 1 here?

Reply:
It's actually from the way I implemented the array size, but we can. I'm assuming it's just for looks, since hopefully the C++ compiler knows its a no-op?